packages/storefront/tests/a11y/specs/accessibility.a11y.ts

```diff
@@ -40,10 +42,13 @@ const gotoUrl = async (page: Page, url: string): Promise<void> => {
 };
 
 const enableDarkMode = async (page: Page): Promise<void> => {
-  const themeBtn = await page.$('.cycle-platform-theme');
-  await themeBtn.click();
-
-  await page.waitForFunction(() => document.body.className === 'dark-mode');
+  const themeBtn = page.locator('.cycle-platform-theme');
+  if (await themeBtn.count()) {
+    await themeBtn.click();
+    await page.waitForFunction(() => document.body.className === 'dark-mode');
+  } else {
```

Comment:
Unnecessary else case. Instead of the condition `themeBtn.count()`, the `themeBtn` has to be defined otherwise the test should fail e.g.
`await expect(themeBtn).toBeDefined`

Reply:
adapted the test.